docs/Configuration.md

```diff
@@ -10,6 +10,25 @@ Each section in `backend_config.ini` is documented below.
 Restart the `zm-rpcapi` and `zm-testagent` daemons to load the changes
 made to the `backend_config.ini` file.
 
+## RPCAPI section
+
+Available keys: `enable_batch_jobs`, `enable_add_api_user`.
```

Comment:
To be consistent, shouldn't the first key be "enable_add_batch_jobs"? It will not affect running batch jobs.

Reply:
Actually this enable both `add_batch_job` and `get_batch_job_result` as I don't see a use case for the second one if the first is disabled.

Reply:
There is no reason to disable method `get_batch_job_result`. You can want to see that even if you have disabled `add_batch_job`.